### PR TITLE
DTGB-865: Fix undefined variables

### PR DESCRIPTION
**Pull request by MPParsley**
## Description
Fix undefined variables.

### CODE REVIEW

File: source/js/checkbox_filter.bindings.js [hunk 1]
```diff
@@ -8,19 +8,19 @@
   Drupal.behaviors.gentBaseLoadCheckboxFilters = {
     attach: function (context, settings) {
       $('.checkbox-accordion', context).once('checkbox-accordion').each(function () {
-        if (Accordion) { // eslint-disable-line no-undef
+        if (typeof Accordion == "undefined") {
```
Comment (zero2one) [inline source/js/checkbox_filter.bindings.js]: !== undefined?

File: source/js/checkbox_filter.bindings.js [hunk 2]
```diff
@@ -8,19 +8,19 @@
...
           for (let i = selected.length; i--;) {
-            new Accordion(selected[i]); // eslint-disable-line no-undef
+            new Accordion(selected[i]);
           }
         }
       });
 
       $('.checkbox-filter-dynamic', context).once('checkbox-filter-dynamic').each(function () {
-        if (CheckboxFilterDynamic) { // eslint-disable-line no-undef
+        if (typeof CheckboxFilterDynamic == "undefined") {
```
Comment (zero2one) [inline source/js/checkbox_filter.bindings.js]: !== undefined?

**Review by zero2one — CHANGES_REQUESTED** [review]

File: source/js/checkbox_filter.bindings.js (hunk 1 above)
Comment (MPParsley) [reply source/js/checkbox_filter.bindings.js]:
```suggestion
        if (typeof Accordion === "undefined") {
```

File: source/js/checkbox_filter.bindings.js (hunk 2 above)
Comment (MPParsley) [reply source/js/checkbox_filter.bindings.js]:
```suggestion
        if (typeof CheckboxFilterDynamic === "undefined") {
```

File: source/js/checkbox_filter.bindings.js [hunk 3]
```diff
@@ -8,19 +8,19 @@
   Drupal.behaviors.gentBaseLoadCheckboxFilters = {
     attach: function (context, settings) {
       $('.checkbox-accordion', context).once('checkbox-accordion').each(function () {
-        if (Accordion) { // eslint-disable-line no-undef
+        if (typeof Accordion === "undefined") {
```
Comment (MPParsley) [inline source/js/checkbox_filter.bindings.js]:
```suggestion
        if (typeof Accordion != "undefined") {
```

File: source/js/checkbox_filter.bindings.js [hunk 4]
```diff
@@ -8,19 +8,19 @@
...
           for (let i = selected.length; i--;) {
-            new Accordion(selected[i]); // eslint-disable-line no-undef
+            new Accordion(selected[i]);
           }
         }
       });
 
       $('.checkbox-filter-dynamic', context).once('checkbox-filter-dynamic').each(function () {
-        if (CheckboxFilterDynamic) { // eslint-disable-line no-undef
+        if (typeof CheckboxFilterDynamic === "undefined") {
```
Comment (MPParsley) [inline source/js/checkbox_filter.bindings.js]:
```suggestion
        if (typeof CheckboxFilterDynamic != "undefined") {
```

**Review by zero2one — APPROVED** [review]

**Pull request merged**